.github/workflows/linters.yml

```diff
@@ -0,0 +1,20 @@
+name: Linters

```

Comment:
- ### These inline comments are for README file:
- [ ] Kindly update your readme for your current project:
![image](https://user-images.githubusercontent.com/62705947/214885720-95733c7a-cb56-438d-88c0-9e21cd32e7e8.png)
![image](https://user-images.githubusercontent.com/62705947/214885784-ad0e99c2-a5c7-4f58-bc24-3cf8c4558df0.png)